settings.py

```diff
@@ -1 +1 @@
-"""Settings for the status NApp."""
+"""Settings for the status NApp. """
```

Comment:
You just have to undo this to fix linter tests again.

Reply:
I fixed the issue.